conformance/utils/http/http.go

```diff
@@ -229,3 +232,26 @@ func CompareRequest(cReq *roundtripper.CapturedRequest, cRes *roundtripper.Captu
 	}
 	return nil
 }
+
+// Get User-defined test case name or generate from expected response to a given request.
+func (er *ExpectedResponse) GetTestCaseName(i int) string {
```

Comment:
Tiny nit: Would expect this to be called "TestCaseName":
```suggestion
func (er *ExpectedResponse) TestCaseName(i int) string {
```

(https://go.dev/doc/effective_go#Getters)

Reply:
Had kept it "GetTestCaseName" as had added a new field in struct ExpectedResponse "TestCaseName" to carry the user given test case name. Will change the field name to "UsrTestCaseName" and the method name to "TestCaseName"

Reply:
Good point, I like your original approach in that case with the field named "TestCaseName" and the func named "GetTestCaseName"